### DISCUSSION TITLE
Duplicate IDs and 4.1.1 Parsing (Obsolete and removed)

### DISCUSSION THREAD

**Discussion by TheUntamed** [w3c/wcag#3531] [Q&A] [answered]
Due to 4.1.1 Parsing being removed with WCAG 2.2, does that mean things like duplicate IDs are not a violation of the WCAG anymore as well?
Or is that one still covered somewhere else?

**Comment by Aircl0wn** [comment, marked as answer]
It depends on the effect this has on the page. If the duplicate IDs cause an element to get an invalid name because the value is overwritten later in the DOM, this could cause a fail under SC 1.3.1 or SC 4.1.2.

SC 4.1.1 got removed because assistive technology no longer relies on Parsing the DOM as browsers are now handling that and exposing end results through the accessibility tree.

**Comment by patrickhlauke** [comment]
> It depends on the effect this has on the page.

Exactly this.
